src/WebForms/UI/Html32TextWriter.cs

```diff
@@ -0,0 +1,726 @@
+//MIT license
+
+using System.Collections;
+using System.Diagnostics;
+using System.Globalization;
+using System.Text;
+using System.Web.UI.WebControls;  // 
+using System.Web.Util;
+
+namespace System.Web.UI;
+
+public class Html32TextWriter : HtmlTextWriter {
```

Comment:
let's not bring this in now. Maybe use a global alias `global using Html32TextWriter = System.Web.UI.HtmlTextWriter`